### PR TITLE
records: update metadata for files async

### LINKED ISSUE #205: Implement _files field handling

### PR DESCRIPTION
**Pull request by Glignos**
* closes #205

### CODE REVIEW

**Comment by ntarocco** [comment]
Blocked by: https://github.com/inveniosoftware/invenio-files-rest/pull/226

File: {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/tasks.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{% include 'misc/header.py' %}
```
Comment (Glignos) [inline {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/tasks.py]: Remove.

File: {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/tasks.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+@shared_task(ignore_result=True)
+def update_record_files_by_bucket(bucket_id):
+    """Given a bucket id, dump its files in the record metadata."""
+    record_bucket = \
+        RecordsBuckets.query.filter_by(bucket_id=bucket_id).first()
+    if record_bucket:
+        record = Record.get_record(record_bucket.record_id)
+        record.files.flush()
+        record.commit()
+        # it might fail with `sqlalchemy.orm.exc.StaleDataError`
```
Comment (lnielsen) [inline {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/tasks.py]: Can you catch the error here to make it explicit and simply do a continue inside - just to make it explicit and avoid that errors are reported in celery which is actually expected behaviour

Comment (ntarocco) [reply {{cookiecutter.project_shortname}}/{{cookiecutter.package_name}}/records/tasks.py]: Will fix in the next PR: https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/200

**Pull request merged**